### PR TITLE
Modified Starter Sample file(c++ and java) version and fixed the bug

### PR DESCRIPTION
**Pull request by sasmita**

### CODE REVIEW

File: kits/starter/c++/VariousSensors.cpp [hunk 1]
```diff
@@ -47,6 +8,11 @@
 #include <iomanip>
 #include <string>
 #include <sstream>
+#include <ssd1327.h>
+#include <ssd1308.h>
+#include <ssd1306.h>
+#include <sainsmartks.h>
+#include <lcm1602.h>
```
Comment (Propanu) [inline kits/starter/c++/VariousSensors.cpp]: Why were these added?

Comment (sasmita) [inline kits/starter/c++/VariousSensors.cpp]: I see. I think, it got added mistakenly while linking other libraries. Please let me know, if you want me to submit a separate pull request.

Comment (Propanu) [inline kits/starter/c++/VariousSensors.cpp]: No need for a separate one, just force push the changes and this PR will reflect it. Thanks!

**Comment by Propanu** [comment]
Fixed & merged. Thanks, but please be more thorough when formatting your patches!

**Comment by sasmita** [comment]
Sure!!
